tsdb/shard.go

```diff
@@ -177,7 +113,8 @@ func (s *Shard) WritePoints(points []Point) error {
 		// marshal the raw data if it hasn't been marshaled already
 		if p.Data() == nil {
 			// this was populated earlier, don't need to validate that it's there.
-			data, err := s.series[p.Key()].measurement.fieldCodec.EncodeFields(p.Fields())
+			warn(p.Name(), s.measurementFields)
```

Comment:
Bah, this is what happens when you rush to commit right before running out for a fun Saturday evening ;)

> On May 23, 2015, at 6:28 PM, Ben Johnson <notifications@github.com> wrote:
> 
> In tsdb/shard.go:
> 
> > @@ -177,7 +113,8 @@ func (s *Shard) WritePoints(points []Point) error {
> >  		// marshal the raw data if it hasn't been marshaled already
> >  		if p.Data() == nil {
> >  			// this was populated earlier, don't need to validate that it's there.
> > -			data, err := s.series[p.Key()].measurement.fieldCodec.EncodeFields(p.Fields())
> > +			warn(p.Name(), s.measurementFields)
> Did you mean to include this?
> 
> —
> Reply to this email directly or view it on GitHub.
>